samples/system-test/quickstart.test.js

```diff
@@ -28,30 +28,29 @@ const expectedDatasetId = `my_new_dataset`;
 let datasetId = `nodejs-docs-samples-test-${uuid.v4()}`;
 datasetId = datasetId.replace(/-/gi, `_`);
 
-test.after.always(async () => {
-  try {
-    bigquery.dataset(datasetId).delete({force: true});
-  } catch (err) {} // ignore error
-});
-
-test.beforeEach(tools.stubConsole);
-test.afterEach.always(tools.restoreConsole);
+describe(`Quickstart`, () => {
+  beforeEach(tools.stubConsole);
+  afterEach(tools.restoreConsole);
+  after(async () => {
+    try {
+      bigquery.dataset(datasetId).delete({force: true});
+    } catch (err) {} // ignore error
+  });
 
-test(`quickstart should create a dataset`, async t => {
-  await new Promise((resolve, reject) => {
-    const bigqueryMock = {
-      createDataset: _datasetId => {
-        t.is(_datasetId, expectedDatasetId);
+  it(`quickstart should create a dataset`, async () => {
+    await new Promise((resolve, reject) => {
+      const bigqueryMock = {
+        createDataset: async _datasetId => {
+          try {
+            assert.strictEqual(_datasetId, expectedDatasetId);
 
-        return bigquery
-          .createDataset(datasetId)
-          .then(([dataset]) => {
-            t.not(dataset, undefined);
+            const [dataset] = await bigquery.createDataset(datasetId);
+            assert.notStrictEqual(dataset, undefined);
 
             setTimeout(() => {
```

Comment:
Instead of `setTimeout` use `await new Promise(r => setTimeout(r, 200));`

Reply:
Done [here](https://github.com/googleapis/nodejs-bigquery/pull/248/commits/2d4c9a15d8eed3fca9e693d4ad09cd146d0d0afd)